views/event_registration_requests.py

```diff
@@ -49,4 +50,67 @@ def get_single_registration(id):
 
     return registration.__dict__
 
+def create_event_registration(new_event_registration):
+    """Adds a new event registration dictionary
+
+    Args:
+        event_registration (dictionary): Information about the event_registration
+
+    Returns:
+        dictionary: Returns the event registration dictionary with an event registration id
+    """
+    sql = """
+        INSERT INTO event_registration
+            (event_id, user_id)
+        VALUES
+            ( ?, ?);
+        """
+    
+    sql_values = (new_event_registration['event_id'], new_event_registration['user_id'])
+    
+    create_resource(sql, sql_values, new_event_registration)
+    
+    return new_event_registration
+
+
+def delete_event_registration(id):
+    """Deletes a single event_registration
+
+    Args:
+        id (int): Event_registration id
+    """
+    with sqlite3.connect("./national_park.sqlite3") as conn:
+        db_cursor = conn.cursor()
+
+        db_cursor.execute("""
+        DELETE FROM event_registration
+        WHERE id = ?
+        """, (id, ))
+
+def update_event_registration(id, new_event_registration):
+    """Updates the event_registration dictionary with the new values
+
+    Args:
+        id (int): Event_registration id
+        new_event_registration (dict): Event_registration dictionary with updated values
+    """
+    with sqlite3.connect("./national_park.sqlite3") as conn:
+        db_cursor = conn.cursor()
+
+        db_cursor.execute("""
+        UPDATE event_registration
+            SET
+                event_id = ?,
+                user_id = ?,
```

Comment:
need to remove comma on line 104

Reply:
Good catch, I just updated and pushed my changes.